bin/rdebug-ide

```diff
@@ -119,13 +124,20 @@ if options.int_handler
   # install interruption handler
   trap('INT') { Debugger.interrupt_last }
 end
-  
+
 # set options
 Debugger.keep_frame_binding = options.frame_bind
 Debugger.tracing = options.tracing
 Debugger.evaluation_timeout = options.evaluation_timeout
 Debugger.catchpoint_deleted_event = options.catchpoint_deleted_event || options.rm_protocol_extensions
 Debugger.value_as_nested_element = options.value_as_nested_element || options.rm_protocol_extensions
 
-Debugger.debug_program(options)
+if options.attach_mode
+  Debugger::MultiProcess::pre_child(options)
 
+  if Debugger::FRONT_END == "debase"
+    Debugger.enable_trace_points
```

Comment:
@equivalence1 that would mean that pause will not work without breakpoints in normal mode, but it does